README.md

```
@@ -145,6 +146,43 @@ Run the entire workflow from a single config file:
+
+#### Cockpit Mode
+
+The `check` command supports a `--mode cockpit` flag for integration with monitoring dashboards. In cockpit mode, the output follows the `sensor.report.v1` schema and includes additional artifacts:
+
+```bash
+perfgate check --config perfgate.toml --bench pst_extract --mode cockpit
+```
+
+Cockpit mode artifact layout (single bench):
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)